endpointmanager/pkg/fetcher/fetcher.go

```diff
@@ -83,7 +83,7 @@ func GetListOfEndpointsKnownSource(rawendpts []byte, source string, listURL stri
 		}
 		result = CernerList{}.GetEndpoints(cernerList, listURL)
 	} else if source == "Epic" {
-		epicList, err := convertInterfaceToList(initialList, "Entries")
+		epicList, err := convertInterfaceToList(initialList, "entry")
```

Comment:
I think what we actually want to do here is remove the source "Epic", since it's now the same as "FHIR", and use the "FHIR" logic for Epic